js/src/components/google-combo-account-card/index.js

```diff
@@ -18,7 +18,7 @@ export default function GoogleComboAccountCard( { disabled = false } ) {
 
 	const isConnected = google?.active === 'yes';
 
-	if ( isConnected && scope.glaRequired ) {
+	if ( isConnected && scope.gmcRequired && scope.glaRequired ) {
```

Comment:
Continue on https://github.com/woocommerce/google-listings-and-ads/pull/2644#pullrequestreview-2420088853.

> The original `scope.glaRequired` checking logic is no longer applicable to the current implementation. The conditions below must check both `scope.gmcRequired` and `scope.adsRequired`.

The `scope.glaRequired` is still used, so the problem is not addressed.

https://github.com/user-attachments/assets/9777ec7e-b91d-4cc6-ba97-7993a40d4315

In addition, [this condition](https://github.com/woocommerce/google-listings-and-ads/blob/1eadc097b0aa1c5193658426dc6f8d634fbc90fd/js/src/components/google-combo-account-card/index.js#L25) needs an adjustment as well in order to request full access.

![image](https://github.com/user-attachments/assets/a785ce96-3997-4a5d-bad8-f84c91567153)

Reply:
@eason9487 I've updated the conditions. Can you double check please?